### DISCUSSION TITLE
bugs in distillations/three_digit_mul_dispatch.py

### DISCUSSION THREAD

**Discussion by bhomass** [567-labs/instructor#812] [General]
need these lines
#instructor.from_openai()  <-- comment out
client = instructor.from_openai(OpenAI())

instructions = Instructions(
    name="three_digit_multiply",
    finetune_format="messages",
    include_code_body=True,
    log_handlers=[
        logging.FileHandler("math_finetunes.jsonl"),
    ],
    openai_client=client,   <--- add this!
)

**Comment by jxnl** [comment]
out rn, would love a pr

**Reply by bhomass** [reply]
pull request submitted.
